src/serac/physics/utilities/functional/tests/functional_material_state_test.cpp

```diff
@@ -0,0 +1,292 @@
+// Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include <gtest/gtest.h>
```

Comment:
I imagine you're wanting to make sure that the restart process preserves the order of the data - would it maybe be sufficient to set each point to something simple like the distance from the origin (since I think we have that info within the QFunc)?